classes/kernel.oeclass

```diff
@@ -15,10 +15,14 @@
 ##          DEPENDS_KERNEL_INITRAMFS to acquire the initramfs.
 ## @useflag ramdisk_image_compression Compression of the ramdisk image. Set to
 ##          none by default.
-## @useflag kernel_dtb Name of the compiled Device tree source (dts).
-## @useflag kernel_dtc Whether to use the linux device tree compiler (dtc).
+## @useflag kernel_dtb Name of the (already compiled) Device tree binary (dtb)
```

Comment:
I don't think it was ever meant as a way to provide an already compiled dtb file.  The idea was (AFAIR) to use USE_kernel_dtb to specify the dtb file that will be build by kbuild process.